rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -1277,4 +1277,51 @@ extern "C" RocalTensor ROCAL_API_CALL rocalSlice(RocalContext context,
                                                  RocalOutOfBoundsPolicy policy = RocalOutOfBoundsPolicy::ROCAL_ERROR,
                                                  RocalTensorOutputType output_datatype = ROCAL_FP32);
 
+/*! \brief Performs mean-stddev normalization on images.
```

Comment:
change the description as 
`Performs mean-stddev normalization on input tensors
`